diff --git a/lib/Net/SMTP/SSL.pm b/lib/Net/SMTP/SSL.pm
index 09fd79d..f004e77 100644
--- a/lib/Net/SMTP/SSL.pm
+++ b/lib/Net/SMTP/SSL.pm
@@ -1,9 +1,9 @@
 package Net::SMTP::SSL;
-# $Id: SSL.pm,v 1.1 2004/07/20 03:22:18 cwest Exp $
+# $Id: SSL.pm,v 1.1 2014/10/24 06:10:00 teckardt Exp $
 use strict;
 
 use vars qw[$VERSION @ISA];
-$VERSION = sprintf "%d.%02d", split m/\./, (qw$Revision: 1.1 $)[1];
+$VERSION = sprintf "%d.%02d", split m/\./, (qw$Revision: 1.2 $)[1];
 
 use IO::Socket::SSL;
 use Net::SMTP;
@@ -13,7 +13,7 @@ use Net::SMTP;
 
 no strict 'refs';
 foreach ( keys %Net::SMTP:: ) {
-    next unless defined *{$Net::SMTP::{$_}}{CODE};
+    next unless defined *{"Net::SMTP::$_"}{CODE};
     *{$_} = \&{"Net::SMTP::$_"};
 }
 
